specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,5668 @@
+        },
+        "ring": {
+          "type": "integer",
+          "description": "The ring number of HostPool."
+        },
+        "validationEnvironment": {
+          "type": "boolean",
+          "description": "Is validation environment."
+        },
+        "registrationInfo": {
+          "$ref": "#/definitions/RegistrationInfoPatch",
+          "description": "The registration info of HostPool."
+        },
+        "vmTemplate": {
+          "type": "string",
+          "description": "VM template for sessionhosts configuration within hostpool."
+        },
+        "ssoadfsAuthority": {
+          "type": "string",
+          "description": "URL to customer ADFS server for signing WVD SSO certificates."
+        },
+        "ssoClientId": {
+          "type": "string",
+          "description": "ClientId for the registered Relying Party used to issue WVD SSO certificates."
+        },
+        "ssoClientSecretKeyVaultPath": {
+          "type": "string",
+          "description": "Path to Azure KeyVault storing the secret used for communication to ADFS."
+        },
+        "ssoSecretType": {
+          "enum": [
+            "SharedKey",
+            "Certificate",
+            "SharedKeyInKeyVault",
+            "CertificateInKeyVault"
+          ],
+          "x-ms-enum": {
+            "name": "SSOSecretType",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "The type of single sign on Secret Type."
+        },
+        "preferredAppGroupType": {
+          "enum": [
+            "None",
+            "Desktop",
+            "RailApplications"
+          ],
+          "x-ms-enum": {
+            "name": "PreferredAppGroupType",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "The type of preferred application group type, default to Desktop Application Group"
+        },
+        "startVMOnConnect": {
+          "type": "boolean",
+          "description": "The flag to turn on/off StartVMOnConnect feature."
+        },
+        "publicNetworkAccess": {
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "PublicNetworkAccess",
+            "modelAsString": true
+          },
+          "type": "string",
+          "description": "Enabled to allow this resource to be access from the public network"
+        },
+        "sessionHostConfiguration": {
+          "$ref": "#/definitions/SessionHostConfigurationProperties",
+          "description": "The session host configurations of HostPool."
+        },
+        "sessionHostComponentUpdateConfiguration": {
+          "$ref": "#/definitions/SessionHostComponentUpdateConfigurationProperties",
+          "description": "The session host configuration for updating agent, monitoring agent, and stack component."
+        }
+      }
+    },
+    "SessionHostConfigurationProperties": {
```

Comment:
As off yesterday need the LastUpdated timestamp in session host config

Reply:
@datr93

Reply:
added the property at the same level as shc because we need to query against it.